access/validator.go

```diff
@@ -75,6 +75,35 @@ func (l *NoopLimiter) IsRateLimited(address flow.Address) bool {
 	return false
 }
 
+type PayerBalanceMode int
+
+const (
+	WarnCheck PayerBalanceMode = iota + 1
```

Comment:
when you add the disabled option, you can make it start from `iota` so 0/uninitialized is the default